src/com/shapestone/laptopshoppingandbilling/CustomersSortedByName.java

```diff
@@ -0,0 +1,49 @@
+package com.shapestone.laptopshoppingandbilling;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Scanner;
+
+import com.fasterxml.jackson.core.exc.StreamReadException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DatabindException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class CustomersSortedByName {
+	public static void main(String[] args) throws StreamReadException, DatabindException, IOException {
```

Comment:
why you have written main here ??

We need to have only one main in Driver program